crypto/s2n_tls13_keys.h

```diff
@@ -73,6 +73,7 @@ extern const struct s2n_blob s2n_tls13_label_traffic_secret_iv;
 
 int s2n_tls13_keys_init(struct s2n_tls13_keys *handshake, s2n_hmac_algorithm alg);
 int s2n_tls13_keys_free(struct s2n_tls13_keys *keys);
+int s2n_tls13_derive_binder_key_secret(struct s2n_tls13_keys *keys, struct s2n_psk *psk);
```

Comment:
Nit: Could this just be called s2n_tls13_derive_binder_key? I get that you're getting an early secret as well from this function, so maybe s2n_tls13_derive_binder_key_and_early_secret? I guess I find the phrase "key_secret" to be redundant.